pkg/csi/service/wcp/controller_helper.go

```diff
@@ -347,8 +348,9 @@ func getVMByInstanceUUIDInDatacenter(ctx context.Context,
 	// Get VM by UUID from datacenter.
 	vm, err := dc.GetVirtualMachineByUUID(ctx, vmInstanceUUID, true)
 	if err != nil {
-		return nil, fmt.Errorf("failed to the VM from the VM Instance UUID: %s in datacenter: %+v with err: %+v",
+		log.Errorf("failed to the VM from the VM Instance UUID: %s in datacenter: %+v with err: %+v",
```

Comment:
`failed to the VM` => `failed to find the VM`

Reply:
Done